evalml/tests/automl_tests/test_automl.py

```diff
@@ -610,24 +567,31 @@ def test_automl_tuner_exception(
         y_train=y,
         problem_type="binary",
         tuner_class=RandomSearchTuner,
-        max_iterations=10,
+        max_batches=5,
         optimize_thresholds=False,
+        _automl_algorithm="iterative",
```

Comment:
This was a tricky test where i could not replicate the `NoParamsException` with `DefaultAlgorithm`. I'm leaving this as a candidate to be fixed in #2868.